spec/firebase_test_runner_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'spec_helper'
+
+describe Fastlane::FirebaseTestRunner do
+  let(:default_file) { '/etc/hosts' }
```

Comment:
What does this value has to do with APKs being tested?! 🤔 
Even if it's supposed to be a fake value, it would be less confusing to use something that at least makes a bit of sense given the context where it's used, no? 😅 

Besides, I came on this line to suggest to use a default file name containing special characters, at the very least spaces, so that you can make sure you also test that escaping and shellwords would work as expected as well, even if the APK being tested has spaces.

```suggestion
  let(:default_file) { '/my home/dev/app v1.2.3 Vanilla Release.apk' }
```

Reply:
Addressed in 1b789ce9def6cf2e0f6637519a9e16509fab0bb2 – separated out the escaping checks though.